src/SingleContainer/Worker/NetworkPerspective.Sync.Worker/Program.cs

```diff
@@ -81,9 +71,11 @@ public static void Main(string[] args)
 
 #if !DEBUG
             builder.Services.RemoveHttpClientLogging();
-#else
-            builder.Services.AddNetworkPerspectiveCoreStub(builder.Configuration.GetSection("Infrastructure:Core"));
 #endif
+            if (builder.Configuration.GetValue<bool>("Infrastructure:Core:Stub"))
```

Comment:
```suggestion
            if (builder.Configuration.GetValue<bool>("Infrastructure:Core:UseStub"))
```
So it's more verbose what type is expected